apps/src/music/player/MusicPlayer.ts

```diff
@@ -322,4 +322,22 @@ export default class MusicPlayer {
 
     return samples;
   }
+
+  private validateBpm(bpm: number): number {
+    if (bpm < 1 || bpm > 500) {
```

Comment:
These bounds were chosen some what arbitrarily. We need a non-zero number, otherwise playback breaks, but we could in theory go as fast as we want. In practice, users won't actually be able to set BPM themselves (it'll be automatically set by the project/level), so this validation is just for internal testing.